tests/openvino/test_modeling.py

```diff
@@ -1292,8 +1297,12 @@ def test_compare_to_transformers(self, model_arch):
         self.assertTrue(ov_model.use_cache)
         tokenizer = AutoTokenizer.from_pretrained(model_id, trust_remote_code=model_arch in self.REMOTE_CODE_MODELS)
         tokens = tokenizer("This is a sample output", return_tensors="pt")
-
-        ov_outputs = ov_model(**tokens)
+        if model_arch == "ernie4_5":
+            # As the offical example, ernie4_5 can only accept input_ids
+            # https://huggingface.co/baidu/ERNIE-4.5-0.3B-PT/blob/main/README.md?code=true#L92
+            ov_outputs = ov_model(tokens.input_ids)
+        else:
+            ov_outputs = ov_model(**tokens)
```

Comment:
I'm not sure this is true, checking the code for ernie4.5 causal lm, the forward pass does accept attention mask and other stuff.

Reply:
also the export uses text encoder with position ids, so how come they shouldn't be passed here ?

Reply:
hmm yeah you're correct it's never used in the internal logic, so the torchscript tracer does not pick it up as input.